03.bowling/bowling.rb

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env ruby
+
+require 'debug'
+
+# 引数をとる
+score = ARGV[0]
+scores = score.split(',')
+
+# 数字に変換
+shots = []
+scores.each do |s|
+    if s == 'X' # strike
+        shots << 10
+        shots << 0
+    else
+        shots << s.to_i
+    end
+end
+
+# 2フレームごとに分割する
+frames = []
+shots.each_slice(2) do |s|
+    frames << s
+end
+
+# 10フレーム目は2投もしくは3投でカウント
+if ! frames[10].nil?
```

Comment:
```suggestion
unless frames[10].nil?
```

と書くことができます。`!`を使うにしても`!frames`のようにスペースを空けない書き方が一般的ですね。